deepchecks/utils/plot.py

```diff
@@ -21,8 +21,8 @@
            'create_confusion_matrix_figure', 'colors', 'hex_to_rgba']
 
 
-colors = {'Train': '#00008b',  # dark blue
-          'Test': '#69b3a2',
+colors = {'Train': '#69b3a2',  # green
+          'Test': '#00008b',   # dark blue
```

Comment:
Why?
Seems confusing to users already using the package...

Reply:
@nirhutnik because the "(" is seen more clearly on the green background where the text is wider than the bar (see the middle pair of bars int he example)

Reply:
But this changes all of the graphs... Seems Like an excessive solution. Maybe just using a lighter shade of blue?

Reply:
We talked about it in the daily, no?

Reply:
@nirhutnik the color definitions are global anyway